src/main/rdplugin/assets/js/jobgraph.js

```diff
@@ -417,21 +417,15 @@ jQuery(function () {
 
     self.load = function (id) {
       return jQuery.ajax({
-        url: _genUrl(appLinks.scheduledExecutionWorkflowJson, {
-          id: id
-        }),
+        url: "/api/34/job/" + id + "/workflow",
         method: 'GET',
         contentType: 'json'
         // success:function(data){ }
       });
     };
     self.find = function (group, name) {
       return jQuery.ajax({
-        url: _genUrl(appLinks.menuJobsAjax, {
-          project: project,
-          jobExactFilter: name,
-          groupPathExact: group
-        }),
+        url: "/api/34/" + project + "/jobs?jobExactFilter=" + name + "&groupPathExact=" + group,
```

Comment:
```suggestion
        url: "/api/"+ appLinks.api_version + "/" + project + "/jobs?jobExactFilter=" + name + "&groupPathExact=" + group,
```

Reply:
Hmmm .. how does that work? apiLinks.api_version is not part of the ui-job-graph project either

Reply:
If we can verify that appLinks is getting passed into the javascript somehow, then maybe the correct solution is to bring the _genUrl() function into this codebase.

Reply:
Well -- I don't know why it works, but it does. I have pushed that change on my branch